### ISSUE TITLE
Strings end up with extra quotations in example config

### ISSUE THREAD

**Issue by martin-langhoff** [bitwalker/conform#66]
`config.exs` contains 

```
config :engine,
  host_id: "some uuid",
```

the generated `engine.schema.exs`:

```
    "engine.host_id": [
      commented: false,
      datatype: :binary,
      default: "nice uuid",
      doc: "Provide documentation for engine.host_id here.",
      hidden: false,
      to: "engine.host_id"
    ],
```

when I generate `engine.conf`, it contains what I understand to be spurious double quotes:

```
# Provide documentation for engine.host_id here.
engine.host_id = "nice uuid"
```

And then this seems to be rather wrong:

```
$ mix conform.effective | grep host_id
 engine: [host_id: "\"nice uuid\"", middlewares: [],
```

Not a showstopper, I am cleaning up the spurious double-quotes. But buggy.

**Comment by martin-langhoff** [comment]
`  "conform": {:hex, :conform, "2.0.0"}`

**Comment by liveforeverx** [comment]
@martin-langhoff 👍 I have the same.

**Comment by liveforeverx** [comment]
My old pull request to this problem: #30 
But, another problem, that in new version input isn't sanitized, what I fix with #70

**Comment by bitwalker** [comment]
@liveforeverx #70 is merged, is #30 still something which needs to be addressed, or did #70 cover the issues outlined here?
